.github/workflows/ci.yml

```diff
@@ -1017,7 +1017,7 @@ jobs:
       BACKEND: "postgres"
       PYTHON_MAJOR_MINOR_VERSION: "${{needs.build-info.outputs.default-python-version}}"
       POSTGRES_VERSION: "${{needs.build-info.outputs.default-postgres-version}}"
-      BACKEND_VERSION: "${{needs.build-info.outputs.default-python-version}}"
+      BACKEND_VERSION: "${{needs.build-info.outputs.default-postgres-version}}"
```

Comment:
BACKEND_VERSION has only ben used for coverage as unique id (that's why it was wroking BTW) so maybe we could rename it to UNIQUE_COVERAGE_KEY or something?

Reply:
Oh... I think it is not unique key rather than for some kind of "tags", that how I actually found that something wrong 

![image](https://user-images.githubusercontent.com/3998685/215329648-231a2690-2ce3-472a-bbf0-e6bb8f9df8df.png)